examples/client/Locomotion/src/pages/ActiveRide/index.js

```diff
@@ -220,6 +233,20 @@ const RidePage = ({ mapSettings, navigation }) => {
           BS_PAGE_TO_COMP[currentBsPage] ? BS_PAGE_TO_COMP[currentBsPage]() : null
         }
       </BottomSheet>
+      <RideCanceledPopup
+        isVisible={ridePopup === RIDE_POPUPS.RIDE_CANCELED_BY_DISPATCHER}
+        onCancel={() => {
+          backToMap();
+          cleanRideState();
+          setRidePopup(null);
+        }}
+        onBackToOrder={() => {
+          setRide({});
```

Comment:
why not cleanRideState?

Reply:
because i want to keep the request stop points

Reply:
Do we have the request stop point at this stage? it's after ride dispatched

Reply:
ya it keeps them in the context state

Reply:
I merged a pr that removes it, after ride is active or dispatch the request stop point is not relevant anymore